vyper/venom/ir_node_to_venom.py

```diff
@@ -137,8 +137,7 @@ def _append_jmp(fn: IRFunction, label: IRLabel) -> None:
 
 def _new_block(fn: IRFunction) -> IRBasicBlock:
```

Comment:
Update the return signature now that the function does not return anything, or return the new block anyway?

Reply:
i think didn't get a chance to run the linter yet!